### DISCUSSION TITLE
save Destinations  Affects  other Destinations   and  Source Transformer,It will cover  Or rewrite it   other Destinations   and  Source Transformer

### DISCUSSION THREAD

**Discussion by ourlang** [nextgenhealthcare/connect#6302] [Q&A]
save Destinations  Affects  other Destinations   and  Source Transformer,It will cover  Or rewrite it   other Destinations   and  Source Transformer

 
﻿
Environment (please complete the following information):
﻿
OS: Windows 10
database:derby
Java Distribution/Version
jdk version "17"
Connect Version 4.4
Workaround(s)
Not applicable.

![image](https://github.com/user-attachments/assets/ab2441b0-bbe6-49c1-addb-0c22cc00c95f)

channel is  publish status

May I ask where the configuration issue is or what other problem it is

**Comment by ourlang** [comment]
Occasionally it can also affect   Source Transformer   ，filter  Sometimes this problem also occurs

**Comment by ourlang** [comment]
mirthConnect   Can channel modifications be made and saved during the publishing process?

Will this affect the content of the saved table 'channel' ？

**Comment by ourlang** [comment]
Additional note: This situation can also occur in SQL Server databases.
save Destinations Affects other Destinations and Source Transformer,It will cover Or rewrite it other Destinations and Source Transformer

**Comment by pacmano1** [comment]
@ourlang I suspect you will need to demonstrate repeatability.  

In other words, on a clean install of Mirth Connect running against SQL server, does this occur and how can it be repeated?

**Reply by ourlang** [reply]
I tested a brand new SQL Server database and found that the Transformer's other Destination was occasionally replaced with the same one

Example: Modify the content of sendOperationApply linex. save changes after。
The content of getAnesthesia Assessment Detail linex will become the same as the one modified above. At this time, the occasional Destination is uncertain, and the next time it will be this one

**Reply by ourlang** [reply]
My configuration screenshot is as follows
![image](https://github.com/user-attachments/assets/909535d5-85bc-4675-b57f-67e98e115251)
![image](https://github.com/user-attachments/assets/27ee6be8-eae9-404c-8f92-cb7943f8e5e0)
![Uploading image.png…]()

**Comment by ourlang** [comment]
![image](https://github.com/user-attachments/assets/32cac00a-cb70-40c9-8dd9-08146f1d40e7)

**Comment by ourlang** [comment]
![image](https://github.com/user-attachments/assets/b45b5aa4-f4ff-4537-ad96-c6223c52a47e)

**Comment by ourlang** [comment]
my channels address  [https://github.com/ourlang/excel/blob/master/Winex60-esb-json.xml](https://github.com/ourlang/excel/blob/master/Winex60-esb-json.xml)
